packages/stryker-api/src/logging/LoggerFactoryMethod.ts

```diff
@@ -0,0 +1,12 @@
+import Logger from './Logger';
+
+/**
+ * Get a logger instance. Instance is cached on categoryName level.
+ *
+ * @param   {String} [categoryName] name of category to log to.
+ * @returns {Logger} instance of logger for the category
+ * @static
+ */
+export default interface LoggerFactoryMethod {
```

Comment:
I don't understand why this exists and what it does

Reply:
I changed the description to:

```
* Represents a factory to get loggers by category name.
 * This interface is used to describe the shape of a logger factory method.
```

Is this better?